__tests__/app.test.js

```diff
@@ -524,3 +509,27 @@ describe("GET /api/articles?topic=", () => {
       });
   });
 });
+
+describe("GET api/articles/:article_id?comment_count=[true/any-string]", () => {
+  test("200: Responds with an article object with a comment_count property", () => {
+    return request(app)
+      .get("/api/articles/1?comment_count=true")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.article_id).toBe(1);
+        expect(article.comment_count).toBe(11);
+      });
+  });
+
+  test("200: Responds with an article object without a comment_count property when given any value that is not 'true'", () => {
+    return request(app)
+      .get("/api/articles/1?comment_count=any-value-including-false")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article.article_id).toBe(1);
+        expect(article).not.toHaveProperty("comment_count");
+      });
+  });
+});
```

Comment:
The comment count should just be added to the article object for this task, it should not be optional.